crm_lead_currency/__manifest__.py

```diff
@@ -2,18 +2,18 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
-    'name': 'CRM Lead Currency',
-    'summary': """
+    "name": "CRM Lead Currency",
+    "summary": """
         On leads/opportunities, add the amount in the customer's currency.""",
-    'version': '12.0.1.0.1',
-    'license': 'AGPL-3',
-    'author': 'Camptocamp SA,Odoo Community Association (OCA)',
-    'website': 'https://github.com/OCA/crm',
-    'depends': [
-        'crm',
+    "version": "14.0.1.0.0",
```

Comment:
Version should be bumpet in the migration commit, not in the pre-commit one. the autoformatting should not include any additional hidden change.